app/main.py

```diff
@@ -1,8 +1,23 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people.update({self.name: self})
```

Comment:
Why you need an update when you can do:
1. `Person.people[self.name] = self`
2. `self.people[self.name] = self`
The result will be the same